### PR TITLE
NEXT-00000 - Fix review filter

### PR DESCRIPTION
**Pull request by ghost**
### 1. Why is this change necessary?
- Currently if you select a point amount as filter in a product review section and combine this filter with the review language filter the point amount selection disappears
- This PR fixes this by including the missing points parameter also in the language switch form
- Currently the `replaceSelectors` of the sortby form are different from the others and therefor look odd on a reload, this PR also fixes this form

### 2. What does this change do, exactly?
* Changed `review.html.twig` to include the missing points parameter in the language filter form
* Changed `review.html.twig` to adjust the `replaceSelectors` of sortby filter form to be similar to all other `replaceSelectors`

### 3. Describe each step to reproduce the issue or behaviour.
- Select a point amount ( 1 - 5 ) in a product review section
- Check the review language filter switch

### 4. Please link to the relevant issues (if any).
- /

### 5. Checklist

- [X] I have rebased my changes to remove merge conflicts
- [X] I have written tests and verified that they fail without my change
- [X] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-implement-new-changelog.md) with all necessary information about my changes
- [X] I have written or adjusted the documentation according to my changes
- [X] This change has comments for package types, values, functions, and non-obvious lines of code
- [X] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Storefront/Resources/views/storefront/component/review/review.html.twig [hunk 1]
```diff
@@ -117,11 +123,7 @@
                                                     {% if productReviewCount > 0 %}
                                                         <div class="col-12 col-md-auto product-detail-review-sortby">
                                                             {% set formAjaxSubmitOptions = {
-                                                                replaceSelectors: [
```
Comment (keulinho) [inline src/Storefront/Resources/views/storefront/component/review/review.html.twig:120]: Why did you change the selectors? It seems unrelated to the other change you did?

Comment (akf-bw) [reply src/Storefront/Resources/views/storefront/component/review/review.html.twig:120]: I changed this to match all other replaceSelectors. For some reason the sort filter has these different selectors, which looks odd when updated in the storefront compared to the other filters (points or language).

Do you want to keep this change or should I remove it (maybe open another PR)?

Comment (ghost) [reply src/Storefront/Resources/views/storefront/component/review/review.html.twig:120]: I adjusted the changelog now.

**Comment by leduc92** [comment]
Thanks for your contribution. Your merge request has been merged. 🎉
